lib/console.js

```diff
@@ -54,6 +56,8 @@ function Console(stdout, stderr, ignoreErrors = true) {
   prop.value = createWriteErrorHandler(stderr);
   Object.defineProperty(this, '_stderrErrorHandler', prop);
 
+  this[kCounts] = {};
```

Comment:
Why not use `Map` here? Won't that remove the restriction about symbols?
Or at least `Object.create(null)` to go straight into dictionary mode.